changelog/unreleased/kong/fix-tcp-log-sslhandshake.yml

```
@@ -0,0 +1,3 @@
+message: **tcp-log**: fix an issue that repeated ssl handshake
```

```suggestion
message: "**tcp-log**: fix an issue that repeated ssl handshake"
```

previously, I mislead you. It should have double quotation marks added 😅